### DISCUSSION TITLE
How to access current window from any place?

### DISCUSSION THREAD

**Discussion by EGSP** [tauri-apps/tauri#9643] [Q&A] [answered]
I need to pull reference to window object to function argument. I dont know how to get reference to current window. Im not familiar with rust, but i think i need some global object that holds references to main components of app, like current window
![image](https://github.com/tauri-apps/tauri/assets/34096047/a225fa62-99ca-4829-8b87-8c6662005dc1)
![image](https://github.com/tauri-apps/tauri/assets/34096047/4141eb5f-9ea4-4252-b52c-d64933ceb99e)

**Comment by EGSP** [comment]
Here i found that we need to do DI. But How?
https://github.com/tauri-apps/tauri/discussions/4202

**Comment by EGSP** [comment, marked as answer]
https://www.youtube.com/watch?v=_fu3XWJHolA
https://tauri.app/v1/guides/features/command/#accessing-the-window-in-commands
![image](https://github.com/tauri-apps/tauri/assets/34096047/c6dff209-0cf9-4037-8be5-309ebc7c48ef)

Aaaaaaaaaaaaa, why that info is placed in "Features" - "Calling Rust from the frontend"

**Reply by FabianLars** [reply]
In which section would you expect it?

**Reply by EGSP** [reply]
Window, Window section and cross section with Command. Or Dependency Injection section. But not in Features. For me Window and its workflow is not a feature
